mindmap/mindmap.py

```diff
@@ -187,7 +188,7 @@ def get_file_key(self, anonymous_user_id: str) -> str:
         Returns:
             str: The key (path) to use in S3.
         """
-        block_id = self.location.block_id # pylint: disable=no-member
+        block_id = self.scope_ids.usage_id.block_id
```

Comment:
why are doing this change?

Reply:
These are two different ways to get the `block_id`. For some reason, running the tests using the first way fails.

```python
self = <MindMapXBlock @E99A parent=<Mock name='mock.get()' id='140611555235344'>, name=???, tags=???, display_name=???, is_static=True>
path_prefix = 'test-anonymous-user-id'

    def get_file_key(self, path_prefix: str) -> str:
        """
        Return the key (path) to save and retrieve the file in S3.
    
        Args:
            path_prefix (str): The path prefix to use in the key (path).
    
        Returns:
            str: The key (path) to use in S3.
        """
>       block_id = self.location.block_id # pylint: disable=no-member
E       AttributeError: 'MindMapXBlock' object has no attribute 'location'

mindmap/mindmap.py:241: AttributeError
```